api/serializers.py

```diff
@@ -29,14 +30,31 @@ class Meta:
         fields = ("text",)
 
 
+class ChatRoomSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ChatRoom
+        fields = (
+            "text_invite_url",
```

Comment:
Should we name these like `text_channel_url` instead to match the model fields names? I'm also thinking that non-Discord backends might not call them invite URLs, so X_channel_url seems sufficiently generalized?

Reply:
Done.